config/routes.rb

```diff
@@ -35,12 +35,13 @@
     end
 
     resource :disability_compensation_form, only: [] do
+      get 'find_rating_info_pid'
```

Comment:
@johnpaulashenfelter made an earlier comment about the RESTiness of this endpoint that I still agree with. All of your objects, including the model itself, is named `RatingInfo`. Thoughts on renaming as such? e.g. `GET /v0/disability_compensation_form/rating_info`

Reply:
I agree that I don't love this name either @omgitsbillryan and @johnpaulashenfelter.  The name of the service we are calling from EVSS is findRatingInfoPID so I wasn't sure what are standards are and if I needed to keep it consistent.  I did shorten the name for the models/objects, etc because we typically don't use action verbs (get/find).  I am more than happy to rename this to something else.